kaldb/src/main/java/com/slack/kaldb/server/Kaldb.java

```diff
@@ -51,7 +51,7 @@ public Kaldb(Path configFilePath) throws IOException {
     KaldbConfig.initFromFile(configFilePath);
   }
 
-  public static void main(String[] args) throws IOException {
+  public static void main(String[] args) throws Exception {
```

Comment:
This can be reverted back to an `IOException` correct? I believe this was leftover from the previous exception change.

Reply:
Yeah, but once I add the pending changes, this will be an Exception. So, left it in there. Reverted the exception.